recipes/mpfr/run_test.sh

```diff
@@ -0,0 +1,2 @@
+cc -L$PREFIX/lib -I$PREFIX/include -lmpfr -lgmp -Wl,-rpath,$PREFIX/lib $RECIPE_DIR/test.c -o test.o
```

Comment:
Applied this [tweak]( https://github.com/conda-forge/staged-recipes/pull/132/commits/f401b8fb3b2bb744d25da25a784cddcac8ac0bb9 ) to fix the test on Mac.